src/test/tests/rendering/annot_macros.py

```diff
@@ -10,6 +10,12 @@
 #  Defect ID:  None
 #
 #  Mark C. Miller, Thu Nov  7 14:17:54 PST 2024
+#
+#  Modifications:
+#    Kathleen Biagas, Mon April 7, 2025
+#    Removed 'fulldbname' test from allmacros, as it fails when run in any
+#    location other than the nightly regression location.
+#
```

Comment:
wouldn't it be better to fix fulldbname so that it works in all locations?

Reply:
If you look at the bottom of the ticket, we agreed to remove the test.

Reply:
Sorry, didn't see that.